docker/build.sh

```diff
@@ -100,15 +100,15 @@ function make_jar()
         bash ./docker/run-mysql.sh start
     fi
 
-    # Get the last commit hash of the model directory and set it as an environment variable
-    GIT_MODEL_HASH=$(git log -n 1 --pretty=format:%h model)
+    # Get the last commit hash and set it as an environment variable
+    GIT_HASH=$(git log -n 1 --pretty=format:%h)
```

Comment:
not sure if there's any benefit to getting the hash from the model directory. I could be wrong about this though.

Reply:
I would lean towards leaving it alone if it's working fine unless there's a particular reason you need to make this change

Reply:
changed it back